src/System.Linq.Expressions/tests/SequenceTests/SequenceTests.cs

```diff
@@ -734,7 +735,7 @@ public static void ElementInit()
             {
                 ElementInit ei1 = Expression.ElementInit(mi1, new Expression[] { ce1 });
             }
-            catch (ArgumentException e)
+            catch (ArgumentException)
```

Comment:
I'm not familiar with what exactly this is testing, e.g. without examining it more I don't know if this is expecting an ArgumentException or just wants to allow it.  An Assert.Throws would likely make sense, but I'll leave that for @VSadov to look into later... my goal now was just to clean up the build and test output.